FAST-LIO/config/mid360.yaml

```diff
@@ -13,6 +14,7 @@ mapping:
     gyr_cov: 0.1
     b_acc_cov: 0.0001
     b_gyr_cov: 0.0001
+    gps_cov: 0.01                  
```

Comment:
Is this normal? Seems really low, GPS can have errors within ~1 meter

Reply:
Yeah! This value was not being used, removed it now